recipes/configure-logstash-kibana.rb

```diff
@@ -0,0 +1,32 @@
+# Cookbook Name:: mh-opsworks-recipes
+# Recipe:: configure-logstash-kibana
+
+::Chef::Recipe.send(:include, MhOpsworksRecipes::RecipeHelpers)
+
+(private_es_hostname, es_attributes) = node[:opsworks][:layers]['elasticsearch'][:instances].first
+
+stack_name = stack_shortname
+elk_info = get_elk_info
+
+template '/etc/logstash/conf.d/logstash.conf' do
```

Comment:
All I really mean is to add something like: 

```
template '/etc/logstash/conf.d/logstash.conf' do
  owner 'root'
  group 'root'
end
```

I think chef would default to that because the recipes run as `root`, but better to be explicit so we know how the file will be chown'd in the end.